week-1/K-functions-parameters/exercise2.js

```diff
@@ -1,4 +1,6 @@
-// Declare your function first
+function divide(a, b) {
+    return 3 / 4;
```

Comment:
See above comment and think about how you could amend this function

Reply:
Done. Thanks again!